react-main/src/app/screens/Game/index.js

```diff
@@ -1,17 +1,64 @@
 import React, { Component } from 'react';
 
 import Board from './components/Board';
+import { calculateWinner } from './utils';
 
 class Game extends Component {
+  state = {
+    history: [{ squares: Array(9).fill(null) }],
+    stepNumber: 0,
+    xIsNext: true
+  };
+
+  handleClick = i => {
+    const { history, xIsNext, stepNumber } = this.state;
+    const current = history.slice(0, stepNumber + 1);
```

Comment:
Here you are getting history items from 0 to step number+1. I think you wanted to get only the stepNumber item.